python/ql/lib/semmle/python/pointsto/PointsTo.qll

```diff
@@ -1312,7 +1312,7 @@ module InterProceduralPointsTo {
    * `var = ...; foo(); use(var)`
    * Where var may be redefined in call to `foo` if `var` escapes (is global or non-local).
    */
-  pragma[noinline]
+  pragma[inline]
```

Comment:
This one was surprising to me, but I assume the performance tests show that it's a net improvement.

Reply:
It's been a while, so I don't really remember.  
But I suppose it was an improvement.